pkg/tenant/api/v1/client.go

```diff
@@ -508,6 +508,25 @@ func (s *APIv1) TenantProjectCreate(ctx context.Context, id string, in *Project)
 	return out, nil
 }
 
+func (s *APIv1) TenantProjectStats(ctx context.Context, id string) (out []*StatValue, err error) {
+	if id == "" {
+		return nil, ErrTenantIDRequired
+	}
+
+	path := fmt.Sprintf("v1/tenant/%s/projects/stats", id)
```

Comment:
```suggestion
	path := fmt.Sprintf("/v1/tenant/%s/projects/stats", id)
```

I guess this worked without the `/`? I think we should add it back in for consistency.